Makefile

```
@@ -10,7 +10,7 @@ INCLUDE_MLKEM = -I mlkem
 INCLUDE_RANDOM = -I randombytes
 INCLUDE_NISTRANDOM = -I test/nistrng
 CFLAGS += -Wall -Wextra -Wpedantic -Wmissing-prototypes -Wredundant-decls \
-  -Wshadow -Wpointer-arith -O3 -fomit-frame-pointer -pedantic \
+  -Wshadow -Wpointer-arith -O3 -fomit-frame-pointer -pedantic -Wno-unknown-pragmas \
```

```suggestion
  -Wshadow -Wpointer-arith -Wno-unknown-pragmas -O3 -fomit-frame-pointer -pedantic \
```
Let's keep the warning flags together. 
Otherwise this looks good to me!